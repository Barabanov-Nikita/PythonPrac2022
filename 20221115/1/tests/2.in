@objcount
class C:
    def __init__(self, val=1):
        print("__init__")
        self.val = val

    def __del__(self):
        print("__del__")

c, d, e = C(), C(2), C(3)
print(C.counter)
del c
print(C.counter)