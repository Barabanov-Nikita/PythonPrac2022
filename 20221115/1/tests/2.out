__init__
__init__
__init__
3
__del__
2
__del__
__del__
